BUILDING.md

```diff
@@ -317,7 +259,7 @@ sqlcipher'). On Windows unfortunately it's a bit more difficult: You'll have
 to download and compile the code as described on the
 [SQLCipher website](https://www.zetetic.net/sqlcipher/) before you can proceed.
 
-If SQLCipher is installed, simply follow the standard instructions for your
+If SQLCipher is installed,<br>simply follow the standard instructions for your
```

Comment:
Same as above.  Seems like a strange place to add a line break?

Reply:
I actually wondering this (it's really personal thing), but in a FHD environment,
I decided that a line break here would be better for readability. 🤔  (Same as above)